erigon-lib/kv/tables.go

```diff
@@ -370,6 +370,7 @@ const (
 
 	// Polygon Bridge
 	PolygonBridgeEvents = "PolygonBridgeEvents" // bridge event.ID -> event
+	PolygonBridgeMap    = "PolygonBridgeMap"    // bridge blockNum -> startEventID, endEventID
```

Comment:
naming: "BridgeMap" is a bit too generic - map for what? I would suggest to stick to the naming that already exists in `BorEventNums` - so `PolygonBridgeEventNums`. I mentioned this previously but I'm not sure if we need new names for these tables given we are using them in a new separate database. If you decide to keep these new `Polygon*` names then let's move this variables to the `polygon/bridge` package instead of here.

data format: why do we need to have a mapping of "blockNum->startEventID,endEventID"? It feels like the end is unnecessary? can we stick to the existing `BorEventNums` mapping of "blockNum->startEventID" or is there a reason for this new logic?